### PR TITLE
added rev overflow link button and added rev overflow points to cart

### PR DESCRIPTION
**Pull request by sanbornshane9**

### CODE REVIEW

**Comment by RadioVoice** [comment]
Looks clean to me, do you have unit tests for the component that show the right html appearing in different scenarios?

File: rss-client/src/app/Account/components/account-settings-page/account-settings-page.component.ts [hunk 1]
```diff
@@ -131,15 +135,31 @@ export class AccountSettingsPageComponent implements OnInit {
...
+      }
+      else{
+        this.myAccount.accTypeId = this.accounts[2].accTypeId;
+        this.myAccount.userId = this.userservice.userPersistance().userId;
+      }
+      index = 2;
+    }
+    if(this.accounts[index] !== undefined && !this.accounts[index] !== null){
+      console.log("hit");
+      this.accountService.createAccount(this.myAccount).subscribe((res) => {});
```
Comment (euruswagner) [inline rss-client/src/app/Account/components/account-settings-page/account-settings-page.component.ts:161]: What is supposed to happen if this if statement fails. Is it okay that nothing happens or should there be an alert for the user?

File: rss-client/src/app/Account/components/account-settings-page/account-settings-page.component.ts [hunk 2]
```diff
@@ -131,15 +135,31 @@ export class AccountSettingsPageComponent implements OnInit {
...
       this.myAccount.userId = this.userservice.userPersistance().userId;
+      index = 1;
     } else if (event == 'Bug') {
       this.myAccount.accTypeId = this.accounts[0].accTypeId;
       this.myAccount.userId = this.userservice.userPersistance().userId;
+      index = 0;
+    }
+    else if (event == 'Overflow'){
+      if(this.accounts[2] == undefined || this.accounts[2] == null){
+        alert("There is not a Rev Overflow account linked with your user");
```
Comment (euruswagner) [inline rss-client/src/app/Account/components/account-settings-page/account-settings-page.component.ts:151]: Where should a user go to link a Rev Overflow account to their user account on RSS?

**Comment by Garen7** [comment]
@sanbornshane9 merge conflict, pull from master and dev and fix the conflicts, if you need help with the conflicts I can fix it real quick

**Pull request closed without merging**